src/Entity/AlertBannerEntity.php

```diff
@@ -149,7 +149,8 @@ public function preSave(EntityStorageInterface $storage) {
     }
 
     // Regenerate a JS token for the updated alert banner.
-    if ($this->get('status')->value) {
+    $status = (bool) $this->get('status')->value;
```

Comment:
We force a bool type here as previous it would either be `true/false` or `"1"/"0"`, this makes it less loose